pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -564,114 +590,100 @@ func (service *SecurityPolicyService) buildRuleAppliedGroupByRule(obj *v1alpha1.
 	return &ruleAppliedGroup, ruleAppliedGroupPath, nil
 }
 
-func (service *SecurityPolicyService) buildRuleSrcGroup(obj *v1alpha1.SecurityPolicy, rule *v1alpha1.SecurityPolicyRule, idx int) (*model.Group, string, error) {
-	var ruleSrcGroupName string
-	sources := rule.Sources
-	ruleSrcGroupID := fmt.Sprintf("sp_%s_%d_src", obj.UID, idx)
-	if len(rule.Name) > 0 {
-		ruleSrcGroupName = fmt.Sprintf("%s-src", rule.Name)
+func (service *SecurityPolicyService) buildRulePeerGroupID(obj *v1alpha1.SecurityPolicy, ruleIdx int, isSource bool) string {
+	if isSource == true {
+		return fmt.Sprintf("sp_%s_%d_src", obj.UID, ruleIdx)
```

Comment:
For this patch, we don't want to change groupID with hash value. 
Moreover, we need to generate group peer contents hash if we want to add hash value for groupID, instead of passing rule obj hash.